cmd/querybuilder.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/snapp-incubator/crafting-table/internal/querybuilder"
+)
+
+var dialect string
+var table string
+
+var querybuilderCmd = &cobra.Command{
+	Use: "qb",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			log.Fatalln("needs a filename and dialect")
+		}
+		filename := args[0]
```

Comment:
* Fix parameter name convention 
* It's better to add a new flag and force a user to fill it.